e2e/acceptance/crate.spec.ts

```diff
@@ -245,4 +245,14 @@ test.describe('Acceptance | crate page', { tag: '@acceptance' }, () => {
 
     await expect(page).toHaveURL('/crates/nanomsg/settings');
   });
+
+  test('keywords are shown when navigating from search', async ({ page, msw }) => {
+    loadFixtures(msw.db);
+
+    await page.goto('/search?q=nanomsg');
+    await page.getByRole('link', { name: 'nanomsg', exact: true }).click();
+
+    await expect(page).toHaveURL('/crates/nanomsg');
+    await expect(page.locator('[data-test-keyword]')).toBeVisible();
+  });
```

Comment:
There's another case that would cause the same issue: navigating from dependencies that have more than one dependency (as this will query with `?ids=`). We would need to expand the fixtures to have at least two dependencies. Let me know if we'd like to include this as well.

Reply:
probably fine as-is. if anything, IMHO we shouldn't extend the hardcoded fixtures and instead setup a dedicated scenario for the test. I would prefer to get rid of the fixtures in general.